src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-apprunner.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: AWS Apprunner integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS Apprunner monitoring integration: what data it reports, and how to enable it."
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for reporting your application and AWS App Runner container data to New Relic observability platform. This document explains the integration's features, how to activate it, and what data can be reported.
+
+## Features
```

Comment:
Can we come up with a stronger set of "features" focused on user value and the specific audience?